syntax/import.go

```diff
@@ -147,8 +148,13 @@ func importURL(url string) (rel.Value, error) {
 		if err != nil {
 			return nil, err
 		}
-
-		return bytesValue("", data), nil
+		if exists, val := cache.exists(url); exists {
+			return val, nil
+		}
+		// Pass space as file path as it is remote url and can't be found in local env
+		val := bytesValue("", data)
```

Comment:
Sorry I just read the master branch code and realized we should use `NoPath` here instead of empty string.

Reply:
```suggestion
		val := bytesValue(NoPath, data)
```

Reply:
Updated.